diff --git a/packages/b/b.0.0.1/opam b/packages/b/b.0.0.1/opam
new file mode 100644
index 0000000..3697498
--- /dev/null
+++ b/packages/b/b.0.0.1/opam
@@ -0,0 +1,13 @@
+opam-version: "2.0"
+synopsis: "Synopsis"
+description: "Description"
+maintainer: "Maintainer"
+license: "Apache-2.0"
+homepage: "https://github.com/ocurrent/opam-repo-ci"
+bug-reports: "https://github.com/ocurrent/opam-repo-ci/issues"
+dev-repo: "git+https://github.com/ocurrent/opam-repo-ci.git"
+doc: "https://ocurrent.github.io/ocurrent/"
+build: []
+depends: [
+  "a-1" {>= "0.0.1"}
+]
diff --git a/packages/b/b.0.0.2/opam b/packages/b/b.0.0.2/opam
new file mode 100644
index 0000000..a47aabf
--- /dev/null
+++ b/packages/b/b.0.0.2/opam
@@ -0,0 +1,15 @@
+opam-version: "2.0"
+synopsis: "Synopsis"
+description: "Description"
+maintainer: "Maintainer"
+author: "Author"
+license: "Apache-2.0"
+homepage: "https://github.com/ocurrent/opam-repo-ci"
+bug-reports: "https://github.com/ocurrent/opam-repo-ci/issues"
+dev-repo: "git+https://github.com/ocurrent/opam-repo-ci.git"
+doc: "https://ocurrent.github.io/ocurrent/"
+unknown-field : "Unknown"
+build: []
+depends: [
+  "a-1" {>= "0.0.2"}
+]
diff --git a/packages/b/b.0.0.3/opam b/packages/b/b.0.0.3/opam
new file mode 100644
index 0000000..5bab63e
--- /dev/null
+++ b/packages/b/b.0.0.3/opam
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+synopsis: "Synopsis"
+description: "Description"
+maintainer: "Maintainer"
+author: "Author"
+license: "Apache-2.0"
+homepage: "https://github.com/ocurrent/opam-repo-ci"
+bug-reports: "https://github.com/ocurrent/opam-repo-ci/issues"
+dev-repo: "git+https://github.com/ocurrent/opam-repo-ci.git"
+doc: "https://ocurrent.github.io/ocurrent/"
+build: []
+depends: [
+  "a-1" {< "0.0.2"}
+]
+conflict-class: [
+  "ocaml-host-arch"
+]
+pin-depends: [ "foo.~dev" "git+https://github.com/bar/foo" ]
diff --git a/packages/system-b/system-b.0.0.1/opam b/packages/system-b/system-b.0.0.1/opam
new file mode 100644
index 0000000..0808e56
--- /dev/null
+++ b/packages/system-b/system-b.0.0.1/opam
@@ -0,0 +1,14 @@
+opam-version: "2.0"
+synopsis: "Synopsis"
+description: "Description"
+maintainer: "Maintainer"
+author: "Author"
+license: "Apache-2.0"
+homepage: "https://github.com/ocurrent/opam-repo-ci"
+bug-reports: "https://github.com/ocurrent/opam-repo-ci/issues"
+dev-repo: "git+https://github.com/ocurrent/opam-repo-ci.git"
+doc: "https://ocurrent.github.io/ocurrent/"
+build: []
+depends: [
+  "a-1" {>= "0.0.2"}
+]
